dialogue/TownNPCs/ShovelGuy/ShovelGuy2.json

```diff
@@ -21,7 +21,7 @@
     ],
     "pl": [
       "Już pracowałem nad tym przez jakiś czas, i nie wygląda na to, że skończę w najbliższym czasie.",
-      "Mogę widzieć wiele ludzi mijając mnie, ale ciągle tu kopię przez tyle dni.",
+      "Mogę widzieć wiele ludzi mijających mnie, ale ciągle tu kopię przez tyle dni.",
```

Comment:
"Widzę wielu ludzi mijających mnie, odkąd zacząłem tu kopać kilka dni temu." I think this sentence sounds a bit more natural with these few changes